camp/TOFSensor/src/main/java/frc/robot/subsystems/Spinner.java

```diff
@@ -1,68 +1,46 @@
-// ************************************************************
-// Bishop Blanchet Robotics
-// Home of the Cybears
-// FRC - First Age - 2026
-// File: Spinner.java
-// ************************************************************
-
-// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
 // Copyright (c) FIRST and other WPILib contributors.
 // Open Source Software; you can modify and/or share it under the terms of
 // the WPILib BSD license file in the root directory of this project.
 
 package frc.robot.subsystems;
 
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants;
 
-import com.revrobotics.spark.SparkMax;
-import com.revrobotics.spark.SparkLowLevel.MotorType;
-import com.revrobotics.spark.SparkBase.PersistMode;
-import com.revrobotics.spark.SparkBase.ResetMode;
-import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
-import com.revrobotics.spark.config.SparkMaxConfig;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkRelativeEncoder;
+import com.revrobotics.SparkPIDController;
+import com.revrobotics.CANSparkBase.ControlType;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
 
-/**The spinner class helps to create motor subsystems which allow the
-* rest of the code to accsess the SparkMax motor controller which
-* controls the neo motor.
-*/
 public class Spinner extends SubsystemBase {
-    private SparkMax motor;
-    private double speed;
-
-
-    /**
-     * Constructer function for a Spinner subsystem
-     */
+    private CANSparkMax Motor;
+
+    private SparkPIDController PidController;
+    private RelativeEncoder encoder;
+    private double kPLeft, kILeft, kDLeft, kIzLeft, kFFLeft, kMaxOutputLeft, kMinOutputLeft, maxRPMLeft, maxVelLeft, minVelLeft, maxAccLeft, allowedErrLeft;
+    private boolean isLeftMotorInverted = false;
+    private double motorReferencePosition = 0.0;
+    private boolean motorsInitalizedForSmartMotion = false;
+    private boolean movementWithinTolerance = false;
+    
+    /** Creates a new ExampleSubsystem. */
   public Spinner() {
-    motor = new SparkMax(Constants.motorCanID, MotorType.kBrushless);
-
-    // Configure motor to break when stopped
```

Comment:
It looks like you removed a bunch of config from this file that was actually needed. I think you need to revert this whole file.